README.md

```diff
@@ -158,14 +152,14 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [ ] Bandwidth
 - [ ] RingCentral
 
-#### 📱 Push 
+#### 📱 Push
 
 - [x] [FCM](https://github.com/novuhq/novu/tree/main/providers/fcm)
 - [x] [Expo](https://github.com/novuhq/novu/tree/main/providers/expo)
 - [ ] [SNS](https://github.com/novuhq/novu/tree/main/providers/sns)
-- [ ] Pushwoosh
+- [ ] [Pushwoosh](https://github.com/novuhq/novu/tree/main/providers/pushwoosh)
```

Comment:
Before adding this change you need to update the branch to the current latest changes of our codebase.